src/thor/isochrone.cc

```diff
@@ -83,8 +83,29 @@ void Isochrone::ConstructIsoTile(const bool multimodal, const unsigned int max_m
     max_distance = max_seconds * 70.0f * kMPHtoMetersPerSec;
   }
 
+  // Form bounding box that's just big enough to surround all of the locations.
+  PointLL center_ll = origin_locations[0].latlng_;
+  AABB2<PointLL> loc_bounds(center_ll.lng(), center_ll.lat(), center_ll.lng(), center_ll.lat());
+
+  for (const auto& loc : origin_locations) {
+    loc_bounds.Expand(loc.latlng_);
+  }
+
+  // Find the location closest to the center.
+  PointLL bounds_center = loc_bounds.Center();
```

Comment:
actually maybe im wrong about this... lines of lat are spaced evenly and even though longitudes arent, the center of two lines of longitude will be the same despite changes in latitude. so yeah, i think im wrong about this?